styles/sass/archive-preview.scss

```diff
@@ -1323,3 +1323,16 @@ figure.preview-overlay-on {
         }
     }
 }
+
+.content-item-preview, .preview-content {
+    .preview-label {
```

Comment:
Why are you namespacing it to `.content-item-preview, .preview-content`? It's hard to work with classes when they are styled differently depending on where in the DOM tree they are inserted.

Reply:
In this case, it is needed as the CSS is heavily nested.